source/documentation/monitoring_apps/metrics.md

```diff
@@ -2,94 +2,56 @@
-
-You must set up Prometheus to request metrics from your region's API endpoint.
-
-1. [Install Prometheus](https://prometheus.io/docs/prometheus/latest/getting_started/) [external link].
-
-1. You must set up a bearer token so the API endpoint can authenticate your Prometheus request. We recommend you use a `bearer_token_file`. Set up an automated cron job to run the following command every 5 minutes:
-
-	```
-	cf oauth-token > /path/to/bearer_token_file.txt
-	```
-
-	where:
-	- `cf oauth-token` is the command that generates a bearer token
-	- `/path/to/bearer_token_file.txt` is the location and name of the `bearer_token_file` used by the Prometheus configuration
-
-1. Configure Prometheus to read the bearer token from the `bearer_token_file.txt`. Refer to the Prometheus [configuration documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#ingress) [external link] for more information.
-
-#### Use Docker to run Prometheus locally
-
-You can set up Prometheus to request metrics from your region's API endpoint by using [Docker](https://www.docker.com/) [external link] to run a local instance of Prometheus.
-
-1. Save the following script as `test-metrics.sh`:
-
-	```
-	#!/usr/bin/env bash
-	set -ue
+- CPU
+- RAM
+- disk usage data
+- memory usage data
+- app crashes
+- app requests
+- app response times
 
-	echo "
-	global:
-	  scrape_interval: 1m
-	  evaluation_interval: 1m
-	  scrape_timeout: 1m
+Before you set up the PaaS Prometheus exporter app, you will need:
 
-	scrape_configs:
-	  - job_name: PaaS
-	    bearer_token: $(cf oauth-token | sed 's/bearer //')
-	    scheme: https
-	    static_configs:
-	      - targets:
-	        - API_ENDPOINT:443
-	" > prometheus.yml
+- a monitoring system to store the metrics with an accompanying metrics endpoint
+- a live Cloud Foundry account assigned to the spaces you want to receive metrics on
 
-	docker run --publish 9090:9090 \
-	           --volume "$PWD/prometheus.yml:/etc/prometheus/prometheus.yml" \
-	           prom/prometheus
-	```
-	where API_ENDPOINT is:
-	- `https://metrics.cloud.service.gov.uk/metrics` for the Ireland region
-	- `https://metrics.london.cloud.service.gov.uk/metrics` for the London region
+We recommend that this Cloud Foundry account:
 
-1. Make the script executable:
+- uses the [`SpaceAuditor` role](/orgs_spaces_users.html#space-auditor) as this role has the minimum permissions needed to meet the requirements of the metrics exporter app
+- is separate to your primary Cloud Foundry account
 
-	```
-	chmod +x test-metrics.sh
-	```
+To set up the PaaS Prometheus exporter app:
 
-1. Execute the script:
+1. Clone the [PaaS Prometheus exporter repository](https://github.com/alphagov/paas-prometheus-exporter).
+1. [Push the PaaS Prometheus exporter app](/deploying_apps.html#deploying-public-apps) to Cloud Foundry without starting the app by running `cf push --no-start prometheus-exporter`.
```

Comment:
When pushing like this, the application would get the same name always `prometheus-exporter`. If multiple tenants push the same application, it would fail for the second one because the domain name is taken.

You can mention they need to put a custom name with `cf push --no-start prometheus-exporter --random-route` or `cf push --no-start prometheus-exporter --hostname prometheus-exporter-my-org-name`.

Then they need to check the domain of the app: https://docs.cloud.service.gov.uk/deploying_apps.html#names-routes-and-domains that would be used later.